src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -428,6 +430,8 @@ public virtual void ConfigureServices(IServiceCollection services)
 
             services.AddTransient<IReleaseDataFileService, ReleaseDataFileService>();
             services.AddTransient<IDataSetFileStorage, DataSetFileStorage>();
+            services.AddScoped<IDataSetUploadRepository, DataSetUploadRepository>();
```

Comment:
We should probably discuss moving to `Scoped` by default with the rest of the dev team cos it sounds like a good idea!